src/bento/caseDetailData.js

```diff
@@ -25,7 +25,7 @@ const leftPanelSubsections = [
       {
         label: 'Arm',
         dataField: 'study_acronym',
-        linkUrl: '/study/{}',
+        linkUrl: '/arm/{}',
```

Comment:
Why don't we try to keep link as uniform instead of linkUrl

Reply:
Sure, I can change that. Is there a uniform name for link on the label? I'm thinking "labelLink".

Reply:
The label should be fine if there is a link associated with it they can always add the link field